docs/blog/2023/announce-mystjs.md

```diff
@@ -0,0 +1,143 @@
+
+Over the years we have seen interest in MyST raise considerably - it allows authors and researchers to express themselves more efficiently than traditional source text languages like LaTeX.
+MyST has also become [a staple in the Python documentation ecosystem](https://www.sphinx-doc.org/en/master/usage/markdown.html), and is the backbone of authoring content in [Jupyter Book](https://jupyterbook.org).
+
+## Bringing MyST to JavaScript with a new partnership
+
+Because the original parser for MyST is written entirely in Python (and strongly tied to Sphinx), we realized that it would be difficult for the impact of this implementation of MyST to make its way out of the Jupyter Book and Python communities.
+
+As a result, we've been exploring how to _extend MyST markdown into JavaScript_, with the goal of integrating with new technical ecosystems and communities.
+We believe this will grow the impact of MyST, and allow for new kinds of workflows around computational narratives.
+The project is called [MyST-JS](https://myst-tools.org/docs/mystjs).
+
+To drive this effort forward, we have [partnered with Curvenote](https://curvenote.com) to upstream their open-source documentation engine into the Executable Books community.
+This effort is based [on their command line interface](https://curvenote.com/docs/cli), which was designed for building technical documentation and computational narratives with JavaScript.
+It will become the backbone of **MyST-JS**, and moving forward will be maintained by the Executable Books community.
+There is still much to do, but you can preview this work already at [js.myst-tools.org](https://myst-tools.org/docs/mystjs).
+
+Here are a few major new steps summarizing this work over the coming months:
+
+### MyST is now a top-level project in Executable Books
+
+We'll treat MyST as another major effort of the Executable Books project, similar to Jupyter Book.
+It will have its own strategy and goals.
+Jupyter Book will remain a consumer of MyST markdown, and it will remain one of many stakeholders in the MyST ecosystem.
+
+:::{admonition} A new website!
+We have set up a dedicated site to describe the MyST project at [myst-tools.org](https://myst-tools.org).
+:::
+
+### MyST will have first-class support in JavaScript
+
+The [MyST-JS project](https://github.com/executablebooks/mystjs) is written entirelly in TypeScript, with the goal of being re-usable across more applications and interfaces.
+It also allows us to leverage the vast ecosystem of JavaScript libraries and frameworks for communication.
+For example, we can use [templating frameworks to export MyST documents to a variety of LaTeX publication formats](https://github.com/myst-templates).
+In the future, we hope to leverage the [unifiedjs project](https://unifiedjs.com/) to make MyST documents even more structured and consumable as data.
```

Comment:
```suggestion
For example, we can use [templating frameworks to export MyST documents to a variety of LaTeX publication formats](https://github.com/myst-templates), and leverage the [unifiedjs project](https://unifiedjs.com/) to make MyST documents even more structured and consumable as data.
```

We are already making considerable use of unifiedjs (e.g. all transformations of the AST, parsing latex, serializing tex), we aren't making use of it yet for markdown-parsing which I think is what the future part referred to. I just tweaked the wording slightly as it is all future facing already.